src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -12,19 +13,17 @@ public FIFORhymer(IntArrayStack intArrayStack) {
 		super(intArrayStack);
 	}
 
-	public final defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
+	public final IntArrayStackInterface temp = new IntArrayStack();
```

Comment:
Wrong choice (correctness more important than efficiency) - IntLinkedList doesn't limit the size